app/addr_prefixes.go

```diff
@@ -37,5 +38,5 @@ func SetAddressPrefixes() {
 		}
 		return fmt.Errorf("unexpected address length %d", n)
 	})
-	//config.Seal()
+	// config.Seal()
```

Comment:
Why this line commented out?

Reply:
@gsk967 do you remember the issue with this? Or can we safely undo it